alarms.tf

```diff
@@ -1,5 +1,5 @@
 resource "aws_cloudwatch_metric_alarm" "alarm_rds_DatabaseConnections_writer" {
-  count               = "${var.cw_alarms ? 1 : 0}"
+  count               = "${var.enabled && var.cw_alarms > 0 ? 1 : 0}"
```

Comment:
`> 0` here doesn't make sense when you're dealing with booleans.

```
${var.enabled && var.cw_alarms ? 1 : 0}
```

Reply:
Removed the >0 check